About Page

kitcath opened the pull request:
Finished a rough draft of the about page

kitcath committed:
About Page

Finished a rough draft of the about page

kellydanma commented on content/pages/about/index.md:
```diff
@@ -2,17 +2,8 @@
 layout: 'aboutlayout'
-heroHeading: 'About Us'
-heroSubHeading: "We've been supporting students since 2010. Join us as we celebrate 10 years of learning!"
+heroHeading: 'Supporting Students Since 2010'
```
Can we remove the capitalization here? 

Eg. `Supporting students since 2010`

Small nitpick I know, it's just to maintain consistency across the website. Capitalization should be used for acronyms (eg. Frequently Asked Questions because the acronym is _FAQ_), but not titles.

kellydanma replied:
Actually nvm, I see why you capitalized it here. 

It's a bit long, can we use something else? The 10 years is implied by the text right underneath. Perhaps:

- `Supporting Students`
- `Supporting Learners`
- etc...

![image](https://user-images.githubusercontent.com/43160110/84451768-f6a30b00-ac21-11ea-9f20-9b6def358457.png)

kellydanma replied:
N.B. It's okay if it doesn't fit into 1 line on mobile, but it would be nice to fit it into one line on desktop.

kellydanma commented on themes/hugo-hero-theme/layouts/pages/aboutlayout.html:
```diff
@@ -4,9 +4,77 @@
+  <br>
+
+  <div class="welcome">
+  <h1 class="title">Our Mission </h1>
```
The title is off centre - can we just exclude it?

![image](https://user-images.githubusercontent.com/43160110/84451932-64e7cd80-ac22-11ea-957c-442f2ced9b5b.png)

kitcath replied:
I decided to put the title " our Mission" as the header of our page.

kellydanma replied:
Lit 🎈

kellydanma commented on data/features.json:
```diff
@@ -1,17 +1,19 @@
-    "description": "New clients recieve an obligation free consultation.",
-    "image": "features/noun_branding_1885335.svg"
+    "title": "Coding",
+    "description": "Build an awesome project with your team over the course of the weekend (and get feedback and mentoring throughout) ",
```
Nitpick here, but this sentence doesn't end with punctuation like the others.

kellydanma commented on themes/hugo-hero-theme/layouts/pages/aboutlayout.html:
```diff
@@ -4,9 +4,77 @@
+  -->
+  <p class="text"> 
+                  
+    Since being founded in 2010, in association with McGill’s Electrical, 
```
This section is pretty long with the sentences being pretty verbose.

I'm wondering if we can either:
1. Shorten this section by employing shorter sentences
2. Cut it down to 1 paragraph and add 3 more `features` to the about section to incorporate the parts we remove

For example, we could add these points:
- `teamwork`
- `learning`
- `hiring` (this is pretty explicit, but that's _okay_, we want sponsors to hire students)

kitcath committed:
Updates to the About Page

I adjusted the page based on your comments. I added more icons and made one paragraph to describe who we are. The page looks a bit empty. Any suggestions to optimize the space we have on the page

kitcath commented:
I adjusted the page based on your comments. I added more icons and made one paragraph to describe who we are. The page looks a bit empty. Any suggestions to optimize the space we have on the page. 

TL;DR:
1- Made the title fit on one line 
2-Made the paragraph shorter 
3-Added more points in the feature.json file

kellydanma committed:
Merge branch 'master' into catherine

kellydanma commented:
It looks good, I'll merge this into `master` and we can always update parts of the web page incrementally :)
